Add jekyll gem: install and configure

Xteff opened the pull request:
![image](https://cloud.githubusercontent.com/assets/8801281/8348847/8c788a5a-1add-11e5-942d-78bd0d9a514d.png)

To test this PR, please:

```
    $ git pull origin chore/configure-jekyll
    $ bundle install
    $ cd fanxhe
    $ jekyll serve

```

After it go to http://localhost:4000 to see the jekyll blog running

Xteff committed:
Add jekyll gem: install and configure

jtapia commented on fanxhe/_config.yml:
```diff
@@ -0,0 +1,14 @@
+  Google search results) and in your feed.xml site description.
+baseurl: "" # the subpath of your site, e.g. /blog/
+url: "http://yourdomain.com" # the base hostname & protocol for your site
+twitter_username: jekyllrb
```
Creo que aqui iria la cuenta de twitter del equipo `Fanxhe`

joiggama replied:
@jtapia No estoy seguro si vamos a nacesitar esa configuración, ya que las plantilla que están haciendo ya tienen estos enlaces de twitter, etc.

jtapia commented:
Se ve bien :+1:

anxhe committed:
Incluír vista de artículo en Jekyll

- Hacer enlaces del menú funcional
- Generar categorías mediante ciclo for
- Inclusión de artículo de prueba
¨

anxhe committed:
Modificar el nombre (blog3.html -> index.html)
